float32 lat
float32 lon
float32 temp
float32 hum
float32 co
float32 co2
float32 nh3
float32 no2
float32 tvoc